OverlayPlugin.Core/MemoryProcessors/Combatant/Common.cs

```diff
@@ -152,6 +170,19 @@ public class Combatant
         public short TransformationId;
         public byte WeaponId;
 
+        // Indirect data from the flags for external use, not included in the JSON for performance concerns
+        [JsonIgnore]
+        public bool IsEnemy;    // if the monster (BattleNPC) is an enemy
+        [JsonIgnore]
+        public bool InParty;    // if the player is in my party (trusted NPCs are not included)
+        [JsonIgnore]
+        public bool InAlliance; // if the player is in my alliance, and not in my party
+        [JsonIgnore]
+        public bool IsFriend;
+
+        [JsonIgnore]
+        public IntPtr Address;
```

Comment:
This is probably useful for debugging as well, so maybe scope the JsonIgnore to not apply when in debug?

```suggestion
#if !DEBUG
        [JsonIgnore]
#endif
        public IntPtr Address;
```